lib/request.js

```diff
@@ -132,6 +132,10 @@ function handleResult (response) {
       return response.json()
     }
 
+    if (contentType.includes('sheet')) {
```

Comment:
acho que vale a pena comparar com `application/vnd.openxmlformats-officedocument.spreadsheetml.sheet`, pois nesse caso sabemos que é mesmo um arquivo de excel
```suggestion
    if (contentType === 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet') {
```